GliaWidgets/Sources/CallVisualizer/ScreenSharing/Style/ScreenSharingViewStyle.Accessibility.swift

```diff
@@ -3,16 +3,20 @@ import Foundation
 extension ScreenSharingViewStyle {
     /// Accessibility properties for ScreenSharingStyle.
     public struct Accessibility: Equatable {
-        /// Flag that provides font dynamic type by setting `adjustsFontForContentSizeCategory` for component that supports it.
+        /// Flag that provides font dynamic type by setting `adjustsFontForContentSizeCategory` 
+        /// for component that supports it.
         public var isFontScalingEnabled: Bool
 
-        ///
-        /// - Parameter isFontScalingEnabled: Flag that provides font dynamic type by setting `adjustsFontForContentSizeCategory` for component that supports it.
+        /// - Parameters:
+        ///   - isFontScalingEnabled: Flag that provides font dynamic type by setting 
+        ///    `adjustsFontForContentSizeCategory` for component that supports it.
```

Comment:
do we need to have additional `///` here?

Reply:
fixed [14839cf](https://github.com/salemove/ios-sdk-widgets/pull/887/commits/14839cfb8b707e3a12197ff4bd96b79b48073da5)